curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-advanced-array-methods-by-building-a-statistics-calculator/66c395a16c3ff33dc6f8e7c0.md

```diff
@@ -0,0 +1,132 @@
+---
+id: 66c395a16c3ff33dc6f8e7c0
+title: Step 26
+challengeType: 0
+dashedName: step-26
+---
+
+# --description--
+
+Change `.sort` to `.toSorted`.
+
+The `toSorted` returns a new array with the elements sorted in ascending order. On the other hand, `sort()` method sorts the elements of array in place.
+
+# --hints--
+
+Your `toSorted` method should have a callback function which takes an `a` and a `b` parameters.
+
+```js
+assert.match(getMedian.toString(), /(array\.toSorted\(\s*\(\s*a\s*,\s*b\s*\)\s*=>|array\.toSorted\(\s*function\s*\(\s*a\s*,\s*b\s*\)\s*\{)/)
+```
```

Comment:
````suggestion
You should use `.toSorted()` instead of `.sort()`.

```js
assert.notMatch(getMedian.toString(), /array\.sort/);
assert.match(getMedian.toString(), /array\.toSorted/);
```
````